### PR TITLE
Resource unpublish

### PR DESCRIPTION
**Pull request by kdimopulu**

### CODE REVIEW

File: staff_features/resources/step_definitions/resource_shared.rb [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  expect(find('#resource_publish_').checked?).to eq true
+end
+
+Given 'the {string} button is displayed' do |button_text|
+  expect(page).to have_css('a', text: button_text)
+end
+
+Then 'the Resource opens on a new tab in the public interface' do
+  expect(page.windows.size).to eq 2
+  expect(page).to have_text "Resource #{@uuid}"
```
Comment (brianzelip) [inline staff_features/resources/step_definitions/resource_shared.rb:21]: Can we be more specific here to test for the public url pattern, something like `#{PUI_URL}/repositories/#{REPO_ID}/resources/#{RESOURCE_ID}`?

**Review by brianzelip — CHANGES_REQUESTED** [review]
@kdimopulu see note below about staff_features/resources/step_definitions/resource_shared.rb

File: staff_features/resources/step_definitions/resource_edit.rb [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+end
+
+Then 'the Resource has one Language with the original values' do
+  visit "#{STAFF_URL}/resources/#{@resource_id}/edit"
+  wait_for_ajax
+
+  languages = all('#resource_lang_materials_ .subrecord-form-wrapper')
+
+  expect(languages.length).to eq 1
+  expect(find('#resource_lang_materials__0__language_and_script__language_').value).to eq 'Australian languages'
```
Comment (balmas) [inline staff_features/resources/step_definitions/resource_edit.rb:65]: 'Australian languages' could be moved to a constant in original_values.rb

**Review by balmas — COMMENTED** [review]
looks good. agree with Brian's requested change.

File: staff_features/resources/step_definitions/resource_shared.rb (hunk 1 above)
Comment (blacksmith-welder) [reply staff_features/resources/step_definitions/resource_shared.rb:21]: Already applied on #36 resource_publish.

**Review by brianzelip — APPROVED** [review]

File: staff_features/resources/step_definitions/resource_edit.rb (hunk 2 above)
Comment (blacksmith-welder) [reply staff_features/resources/step_definitions/resource_edit.rb]: Done

**Review by brianzelip — APPROVED** [review]
👍🏼

**Pull request merged**